__DEFINES/_macros.dm

```diff
@@ -176,11 +176,11 @@
 
 #define ismultitool(A) istype(A, /obj/item/device/multitool)
 
-#define iscrowbar(A) istype(A, /obj/item/tool/crowbar)
+#define iscrowbar(A) istype(A, /obj/item/tool/crowbar) || (istype(A, /obj/item/weapon/switchtool) && istype(A:deployed, /obj/item/tool/crowbar))
```

Comment:
causes #34308

Reply:
what the hell
this also lets you create one-way windows if you crowbar a fulltile rwindow(also infinite plastic)

Reply:
Yes, I saw this too crazy code there to be on base window.